### ISSUE TITLE
EXC_BREAKPOINT: _existingView > countByEnumeratingWithState:objects:count: > frame > horizontalSizeClass > sizeFo...

### ISSUE THREAD

**Issue by jkmassel** [wordpress-mobile/WordPress-iOS#15094]
**Sentry Url**: https://sentry.io/share/issue/ca758ecaa3b84eb798ccf5e5e8a851f2/
**User Count**: 0
**Count**: 42
**First Release**: 15.8.0.2
**First Seen**: 2020-10-06T20:27:37Z
**Last Seen**: 2020-10-13T21:25:47Z
**24 Hours**: 8
**30 Days**: 39
**LogID:** 7738DE90-12E8-4859-9476-C56AE2BB3DDC

```
EXC_BREAKPOINT: _existingView > countByEnumeratingWithState:objects:count: > frame > horizontalSizeClass > sizeFo...
  File "LoginPrologueViewController.swift", line 533, in LoginPrologueViewController.setButtonViewMargins
  File "LoginPrologueViewController.swift", line 116, in LoginPrologueViewController.viewWillTransition
  File "<compiler-generated>", line 114, in LoginPrologueViewController.viewWillTransition
  File "WordPressAuthenticator.swift", line 172, in WordPressAuthenticator.showLogin
  File "WordPressAppDelegate.swift", line 608, in WordPressAppDelegate.showWelcomeScreen
  File "WordPressAppDelegate.swift", line 602, in WordPressAppDelegate.showWelcomeScreenIfNeeded
  File "<compiler-generated>", line null, in WordPressAppDelegate.showWelcomeScreenIfNeeded
  File "Classes/ViewRelated/Blog/Blog List/BlogListViewController.m", line 232, in -[BlogListViewController maybeShowNUX]
  File "Classes/ViewRelated/Blog/Blog List/BlogListViewController.m", line 157, in -[BlogListViewController viewWillAppear:]
  File "main.swift", line 7, in main
...
(78 additional frame(s) were not displayed)
```

**Comment by yaelirub** [comment]
High priority by version. @ScoutHarris, would you please take a look?

**Comment by mindgraffiti** [comment]
I haven't found much of anything so far. I checked out [this hint](https://github.com/wordpress-mobile/WordPress-iOS/issues/12669#issuecomment-568551056) from a previous EXC_BREAKPOINT crash. There are no implicitly unwrapped optionals when moving from `WordPressAuthenticator.showLogin` to the `LoginPrologueViewController.viewWillTransition` method. And an `untouchable view controller` didn't net any helpful search results either.

**Comment by ScoutHarris** [comment]
I coudn't repro this, and like @mindgraffiti I didn't find anything that screamed "I'm going to crash!". But after some poking around, I think I found a possible issue.

The app was crashing on this line:
`buttonViewLeadingConstraint.constant = defaultButtonViewMargin`

That led me to believe that assuming the button view constraints actually exist might be a bad thing. So I've made the constraints optional with [this PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/15232). 🤞 that'll do it!
